app/concepts/todo_list/create.rb

```diff
@@ -0,0 +1,32 @@
+class TodoList::Create < Trailblazer::Operation
+  include Model
+  model TodoList, :create
+
+  contract do
+    property :name
+    property :user_id
+  end
+
+  def process(params)
+    validate(params[:todo_list]) do |f|
+      id, uid = params[:todo_list][:id], params[:todo_list][:user_id]
+      @model = TodoList.find_by_id(id) ||
```

Comment:
You can get rid of the @model call by overriding Trailblazer's `def model!(params); end` method. Create this method inside this operation to override it, then place your `find_or_create` logic in there.